system_tests/espresso_e2e_test.go

```diff
@@ -165,4 +277,74 @@ func TestEspressoE2E(t *testing.T) {
 		return validatedCnt >= msgCnt
 	})
 	Require(t, err)
+
+	hostIo, err := ospgen.NewOneStepProverHostIo(common.HexToAddress(hostIoAddress), builder.L1.Client)
```

Comment:
I think that this test is a little hefty with this staker logic - Can we perhaps make a simple base test and refactor the staker logic out into a second one?

Reply:
Yeah I do think so. But it is not easy because the staker needs those stuffs(L1, L2, validator, sequencer, contracts, and of course, hotshot deployment).